antora.yml

```diff
@@ -69,34 +69,48 @@ asciidoc:
       - page: develop/guide-nodejs/
         reason: Page moved to Redpanda Labs.
     # Data for the home page
-    page-feature-list:
-      - title: 'Quickstart'
-        image: quickstart.png
-        description: 'Learn how to create a basic streaming application.'
+    page-home-intro: Redpanda is a Kafka-compatible streaming data platform built for data-intensive applications. Connect anything with 220+ connectors, process in real time with built-in transformations, and optimize costs with Tiered Storage.
```

Comment:
Redpanda Self-Managed includes:

- Redpanda Enterprise Edition
Requires an xref:get-started:licenses.adoc[Enterprise license], and includes key components such as Redpanda Console, a web-based user interface (UI) for managing and monitoring Redpanda clusters and streaming workloads. While Console is an integral part of Redpanda Cloud, it can also be used as a standalone program as part of a Redpanda Self-Managed deployment. See xref:get-started:licenses.adoc[Enterprise license] to view all Enterprise Edition features.

- Redpanda Community Edition
Requires a xref:get-started:licenses.adoc#redpanda-community-edition[Redpanda Business Source (BSL) license] and includes free core features such as xref:manage:schema-registry.adoc[Schema Registry] and xref:develop:data-transforms.adoc[Data Transforms], which are source-available.
```

Reply:
@Feediver1 please review latest [preview](https://deploy-preview-703--redpanda-docs-preview.netlify.app/current/home/). Enterprise features are not capitalized here because I thought it looked odd to have them capitalized but not transforms or connectors. We can discuss!